File: Eldar Dautović/ridewithme/ridewithme.Model/Models/PoslovniIzvjestaj.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ridewithme.Model.Models
{
    public class PoslovniIzvjestaj
    {
        public int Godina { get; set; }
        public int BrojAdministratora { get; set; }

        public int BrojVoznji { get; set; }

        public int BrojKorisnika { get; set; }

        public int BrojKreiranihKupona { get; set; }

        public int BrojIskoristenihKupona { get; set; }

        public double PrihodiVozaca { get; set; }
    }
}

File: Eldar Dautović/ridewithme/UI/ridewithme_admin/lib/models/poslovni_izvjestaj.dart
// -- poslovni_izvjestaj.dart --
import 'package:json_annotation/json_annotation.dart';

part 'poslovni_izvjestaj.g.dart';

@JsonSerializable()
class PoslovniIzvjestaj {
  int? godina;
  int? brojAdministratora;
  int? brojVoznji;
  int? brojKorisnika;
  int? brojKreiranihKupona;
  int? brojIskoristenihKupona;
  double? prihodiVozaca;

  PoslovniIzvjestaj(
    this.godina,
    this.brojAdministratora,
    this.brojVoznji,
    this.brojKorisnika,
    this.brojKreiranihKupona,
    this.brojIskoristenihKupona,
    this.prihodiVozaca,
  );

  factory PoslovniIzvjestaj.fromJson(Map<String, dynamic> json) =>
      _$PoslovniIzvjestajFromJson(json);

  Map<String, dynamic> toJson() => _$PoslovniIzvjestajToJson(this);
}
